tools/flang1/flang1exe/func.c

```diff
@@ -3967,14 +3967,22 @@ rewrite_calls(void)
         }
         sptr_lhs = memsym_of_ast(A_SRCG(ast));
         if (allocatable_member(sptr_lhs)) {
-          rewrite_deallocate(A_SRCG(ast), false, std);
-          if (!ALLOCG(sptr_lhs) && !ALLOCATTRG(sptr_lhs) &&
-              !POINTERG(sptr_lhs)) {
-            /* Has allocatable members but item itself is not
-             * allocatable nor pointer
-             */
+          /* Has allocatable members but item itself is not
+          * allocatable nor pointer
+          */
```

Comment:
For a comment block, align the asterisks in the same column. See the original code for example.